spinta/backends/mongo.py

```diff
@@ -46,13 +46,15 @@ def __init__(self, connection, id):
 
 @load.register()
 def load(context: Context, backend: Mongo, config: BackendConfig):
-    # Load Mongo component from configuration.
-    pass
+    # Load Mongo client using configuration.
+    backend.name = config.name
+    backend.db_name = config.db_name
+    backend.engine = pymongo.MongoClient(config.dsn)
```

Comment:
Maybe rename `engine` to `mongo`? Engine is more sqlalchemy thing.

Reply:
Renamed.